### PR TITLE
check for log file location in artifacts

### PR DESCRIPTION
**Pull request by ryanfkeepers**
ensure test artifacts have log files

---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :robot: Supportability/Tests

#### Test Plan

- [x] :green_heart: E2E

### CODE REVIEW

**Comment by ryanfkeepers** [comment]
log files are now added to the ci artifacts.  See for example: https://github.com/alcionai/corso/actions/runs/5060181350?pr=3470

File: src/internal/tester/cli.go [hunk 1]
```diff
@@ -32,34 +29,3 @@ func StubRootCmd(args ...string) *cobra.Command {
 
 	return c
 }
-
-func NewContext() (context.Context, func()) {
```
Comment (ryanfkeepers) [inline src/internal/tester/cli.go:36]: moved because I keep forgetting what file these are in and always try to look for `/tester/context.go`

File: src/internal/connector/exchange/restore_test.go
Comment (ashmrtn) [inline src/internal/connector/exchange/restore_test.go:1]: this needs to be moved to a different PR if it's not due to being behind main

**Review by ashmrtn — APPROVED** [review]
So next question, will we be able to tell which log output belongs to which test after we download an artifact?

File: src/internal/connector/exchange/restore_test.go
Comment (ryanfkeepers) [reply src/internal/connector/exchange/restore_test.go:1]: I honestly don't understand this statement.

**Comment by ryanfkeepers** [comment]
> So next question, will we be able to tell which log output belongs to which test after we download an artifact?

@ashmrtn  see: https://github.com/alcionai/corso/pull/3484

File: src/internal/connector/exchange/restore_test.go
Comment (ashmrtn) [reply src/internal/connector/exchange/restore_test.go:1]: oh, sorry, if the changes in this file are not due to the PR being behind main (i.e. some diff weirdness in github) then these changes should be moved to another PR because they're not related to getting logging setup at all
